docs/pages/versions/unversioned/sdk/build-properties.mdx

```diff
@@ -9,15 +9,19 @@ import { ConfigPluginExample } from '~/components/plugins/ConfigSection';
 import { APIInstallSection } from '~/components/plugins/InstallSection';
 import PlatformsSection from '~/components/plugins/PlatformsSection';
 
-**`expo-build-properties`** is a [config plugin](/guides/config-plugins) configuring the native build properties of your `ios/Podfile.properties.json` and `android/gradle.properties` directories during [Expo Prebuild](/workflow/prebuild). This config plugin configures how [Expo Prebuild](/workflow/prebuild) generates the native `ios` and `android` folders and therefore cannot be used with projects that don't run `npx expo prebuild` (bare projects).
+**`expo-build-properties`** is a [config plugin](/guides/config-plugins) configuring the native build properties
```

Comment:
```suggestion
`expo-build-properties` is a [config plugin](/guides/config-plugins) configuring the native build properties
```

The ** outside of backticks don't render. I think its safe to remove them.

Reply:
Agree, didn't spot that, nice catch! 👍